### PR TITLE
feat:builds the options page layout and navigation

### LINKED ISSUE #10: Build the options page layout and Navigation
This is options page layout:
Image

This ticket only asks for the layout and not the content of the views. A simple placeholder text like "This is the block settings view", "This is the Pin Protection view", etc should suffice. The various views will be built separately.

### Acceptance Criteria
- User can navigate in the layout. If user clicks Pin Protection, they should navigate to the screen
- The layout should look like the design
- The navigation sidebar should be fully built with logo

### PR DESCRIPTION
**Pull request by brendagold**
Resolves #10  

### What does this PR do?

- User can navigate in the layout. If the user clicks Pin Protection, they should navigate to the screen

Screenshot 2023-11-01 at 9 57 23 PM

### CODE REVIEW

**Comment by iamstarcode** [comment]
Looks good, well-done Boss

**Review by iamstarcode — DISMISSED** [review]
Looks good

File: src/components/SideNav.tsx
Comment (iamstarcode) [inline src/components/SideNav.tsx:1]: We could put this inline SVG into the assets folder yeah?

Comment (brendagold) [reply src/components/SideNav.tsx:1]: I did that, but the color change wasn't working

Comment (iamstarcode) [reply src/components/SideNav.tsx:1]: Please, what do you mean colour change

**Review by DotCampus-Admin — APPROVED** [review]
LGTM!

File: src/components/SideNav.tsx
Comment (brendagold) [reply src/components/SideNav.tsx:1]: When you hover or click on the nav items, the color changes from black to white. But when I imported the SVG as an external file, the color change did not work.

**Review by DotCampus-Admin — APPROVED** [review]

**Review by Riri202 — COMMENTED** [review]
Welldone Brenda.

**Review by DotCampus-Admin — APPROVED** [review]

**Review by TheoOkafor — APPROVED** [review]

**Pull request merged**